pallets/parachain-system/src/relay_state_snapshot.rs

```diff
@@ -273,4 +277,18 @@ impl RelayChainStateProof {
 		)
 		.map_err(Error::UpgradeRestriction)
 	}
+
+	pub fn read_entry<T>(&self, key: &[u8], fallback: Option<T>) -> Result<T, Error>
```

Comment:
Why not directly expose the functions to read the babe randomness? As we do for all the other values?

Reply:
Then we will need functions for every value we would like to get. This will be at least:
```
read_current_block_randomness
read_one_epoch_ago_randomness
read_two_epochs_ago_randomness
read_epoch_index
```

It is less code to write one `read_entry<T>` which is generic over the successful result value returned.

Reply:
I get what you say, but currently it also requires that you add these values before on the client side :P 

We can add this function, I don't really care, but then please add some docs :)